cmis/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+CMIS Connector
+==============
+
+This module is the base for Odoo modules implementing different integration
+scenario with a CMIS server.
+It allows you to configure a CMIS backend in Odoo.
+
+Installation
+============
+
+No installation required:
```

Comment:
@max3903 Yes, all my changes are proposed upstream but it seems that the lib is no more actively developed...
https://issues.apache.org/jira/browse/CMIS-971 (merged)
https://issues.apache.org/jira/browse/CMIS-972 (pending)
https://issues.apache.org/jira/browse/CMIS-979 (pending)